### PR TITLE
📄 Add examples to README

### PR DESCRIPTION
**Pull request by cowboyd**
## Motivation

Get a good JSR score

## Approach

JSR requires you to have code examples in your README or module doc in order to get a good score, so this adds the very basic usage, and also ignores the `WebSocketData` type since it is an alias and code editors will expand it to its actual type in intelligence.

### CODE REVIEW

File: websocket/README.md [hunk 1]
```diff
@@ -5,3 +5,49 @@ API as an Effection resource. Instead of a fragile, spring-loaded confederation
...
 organizes them for you so that you can consume all events from the server as a
 plain stream that has state-readiness and proper error handling baked in.
+
+To use a websocket import the `useWebSocket()` operation which behaves just like
+the [`WebSocket`](https://developer.mozilla.org/en-US/docs/Web/API/WebSocket)
+constructor.
+
+```ts
+import { main, each } from "effection";
+import { useWebSocket } from "
```
Comment (taras) [inline websocket/README.md:15]: Missing the import here

File: websocket/README.md [hunk 2]
```diff
@@ -5,3 +5,49 @@ API as an Effection resource. Instead of a fragile, spring-loaded confederation
...
+- If the socket recieves an error event, that event's error will be thrown to
+  the current error boundary.
+- The socket is a stream whose items are each `MessageEvents`, the `CloseEvent`
+  of the websocket will be the close event of that stream.
+
+You can also instantiate a websocket separately and pass it along to
+`useWebSocket()`. This is helpful for runtimes such as NodeJS prior to version
+21 that do not have built in support for websocket.
+
+```
```
Comment (taras) [inline websocket/README.md:42]: Missing language here

**Review by taras — APPROVED** [review]

File: websocket/README.md (hunk 2 above)
Comment (cowboyd) [reply websocket/README.md:42]: What language is missing?

Comment (taras) [reply websocket/README.md:42]: just need to add `ts` to the backticks like you have in the previous code example

Comment (cowboyd) [reply websocket/README.md:42]: Ah, right. I thought you meant that there was more to say.

Comment (taras) [reply websocket/README.md:42]: Nope :) sorry, i could have been more explicit

**Pull request merged**